eth/core/precompile/method_test.go

```diff
@@ -18,7 +18,7 @@
 // MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
 // TITLE.
 
-package precompile_test
+package precompile
```

Comment:
@shampoobera can you break this file up into `method_test.go` && `method_validation_test.go`

Reply:
<img width="445" alt="image" src="https://github.com/berachain/polaris/assets/132924353/e0acc443-7f19-442c-b0b8-89eec657a8e2">